src/script/common/c_converter.cpp

```diff
@@ -405,6 +405,16 @@ std::vector<aabb3f> read_aabb3f_vector(lua_State *L, int index, f32 scale)
 	return boxes;
 }
 
+void push_aabb3f_vector(lua_State *L, const std::vector<aabb3f> &boxes, f32 divisor)
+{
+	lua_createtable(L, boxes.size(), 0);
+	u8 i = 1;
```

Comment:
Where does the limit of 255 boxes come from? Is it documented anywhere?

Reply:
`NodeBox::serialize` and `NodeBox::deSerialize` use u16 for array lengths, so I guess using u8 here is wrong. I've now changed it to an int since that's what `lua_rawseti` accepts.